modules/ROOT/pages/hl7-edi.adoc

```diff
@@ -739,7 +734,7 @@ In the HL7 v3.0 connector, the sections that follow list components that have be
 
 === HL7 Message Component
 
-The creation of a message can be done by generating a HL7 Message through DataWeave and then invoking the connector's Write operation.
+The creation of a message can be done by generating a HL7 Message through DataWeave and then invoking the connector's write operation.
```

Comment:
Incorrect fix. Should be `Write` operation -- see https://docs.mulesoft.com/connectors/hl7/hl7-connector-reference#write